pkg/csi/service/osutils/linux_os_utils.go

```diff
@@ -29,6 +29,7 @@ import (
 
 	"github.com/akutz/gofsutil"
 	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/prometheus/common/log"
```

Comment:
Can we use the correct logger? looks like accidentally Prometheus log is used.

Reply:
Sure, I should have noticed that.